example/index.rst

```diff
@@ -0,0 +1,241 @@
+.. Example documentation master file, created by
```

Comment:
this must be a git merge issue(i left the branch un-updated for too long): this file is moved to doc/ folder

Reply:
Removed by 98f1ac577116fde395a47f9c2b2c0755dc05708b